libs/zzz/dm/src/dm/character/character.ts

```diff
@@ -232,13 +232,15 @@ export const charactersDetailedJSONData = Object.fromEntries(
         coreStats: Object.values(raw.ExtraLevel).map(
           ({ Extra }) =>
             Object.fromEntries(
-              Object.values(Extra).map(({ Name, Value }) => [
```

Comment:
Probably time to expand this to a proper block of code with variable names and if statements